tests/integration/actions.go

```diff
@@ -1008,6 +1009,10 @@ func (tr TestRun) unjailValidator(action unjailValidatorAction, verbose bool) {
 	if err != nil {
 		log.Fatal(err, "\n", string(bz))
 	}
+
+	// wait a bit so the tx get included in a block
+	// and packets commited before proceeding
+	time.Sleep(20 * time.Second)
```

Comment:
We need to wait two blocks right? One for the tx to be included, and one for the val set to be committed. Might be worth making this 40 seconds considering the block time you've observed when these tests run in cloud

Reply:
```suggestion
		tr.waitBlocks(action.provider, 2, time.Minute)
```
We can actually do this. Don't know why I didn't think of that right away :D